dev/com.ibm.ws.javaee.ddmodel/resources/com/ibm/ws/javaee/internal/ddmodel/resources/DDModelMessages.nlsprops

```diff
@@ -141,3 +237,23 @@ module.name.not.specified.useraction=Add a moduleName attribute to the web-bnd,
 module.name.invalid=CWWKC2277E: One or more module names on the {1} element are invalid. The invalid module name or names are {0}.
 module.name.invalid.explanation=The binding and extension configuration for web modules must specify a valid moduleName attribute. At least one configuration has specified an invalid module name. The module name must be the name of the archive file that contains the web or EJB application.
 module.name.invalid.useraction=Correct the moduleName attribute in the binding or extension configuration. 
+
+# ADDED:
+# -----------------------------------------------------------------------
+# PARAMETERS:
+#    {0} is the path to a deployment descriptor, including paths to parent archives
+#    {1} is the number of a line within the deployment descriptor
+#    {2} is a DTD public ID value which is present in the descriptor
+# -----------------------------------------------------------------------
+unsupported.descriptor.public.id=CWWCK27782E: The deployment descriptor {0}, at line {1}, specifies unsupported public ID {2}.
```

Comment:
Change "deployment descriptor {0}" to "{0} deployment descriptor"

Please do a global search and replace for this phrase.